examples/upscale_cond.cpp

```diff
@@ -963,8 +963,8 @@ try
        for (int idx=0; idx < points; ++idx) {
            if (node_vs_pressurepoint[idx] != 0) {
                // Receive data
-               double recvbuffer[2+tensorElementCount];
-               MPI_Recv(recvbuffer, 2+tensorElementCount, MPI_DOUBLE, 
+               std::vector<double> recvbuffer(2+tensorElementCount);
+               MPI_Recv(&recvbuffer[0], 2+tensorElementCount, MPI_DOUBLE,
```

Comment:
Just a suggestion: we no longer need to support (very) ancient versions of gcc, so vector::data() is available.